src/entities/tokens.ts

```diff
@@ -110,3 +110,31 @@ export interface RainbowToken extends Asset {
 export interface IndexToken extends Asset {
   amount: string;
 }
+
+export interface ExchangeAsset {
+  address: EthereumAddress;
+  balance: { amount: string; display: string };
+  decimals: number;
+  icon_url: string;
+  id: string;
+  implementations: {
+    [network: string]: { address: EthereumAddress; decimals: number };
+  };
+  isNativeAsset: boolean;
+  isRainbowCurated: boolean;
+  isVerified: boolean;
+  is_displayable: boolean;
+  is_verified: boolean;
```

Comment:
i'm logging these params while searching for assets and i can't find them on the asset objects, am i missing something?